app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,7 @@
     <uses-feature android:name="android.hardware.telephony" android:required="false"/>
 
     <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```

Comment:
This permission already exists in the manifest and can be removed.

Reply:
Indeed, I did not see that the permission was already there 😅